app/config/ApplicationConfig.scala

```diff
@@ -92,6 +92,11 @@ object ApplicationConfig extends ApplicationConfig with ServicesConfig {
     value
   }
 
+  def phase2ChangesToggle: Boolean =  {
```

Comment:
Could just use what is on line 123

Reply:
Yes - we'll give it a go (we had that originally and came across some issues, so we changed it to be the same as others - turned out the issues where nowt to do with it)